lib/Fetcher/FeedFetcher.php

```diff
@@ -233,11 +233,17 @@ protected function buildItem(ItemInterface $parsedItem, ?string $body = null, bo
     {
         $item = new Item();
         $item->setUnread(true);
-        $item->setUrl($parsedItem->getLink());
-        if ($parsedItem->getPublicId() == null) {
+        $itemLink = $parsedItem->getLink();
+        $item->setUrl($itemLink);
+        $publicId = $parsedItem->getPublicId();
+        if ($publicId == null) {
+            // Fallback on using the URL as the guid for the feed item if no guid provided by feed
```

Comment:
Can we log this on an info level? That way it's easier to debug issues with it later

Reply:
Sure. Should have thought of that myself really.

 I'll look up how to do that in the code base and have another PR ready soon for that.

Reply:
Thanks!

Reply:
I think that will do it. I wanted to include the feed URL in the log, but it looks like i don't have convenient access to that at the point I'd need to log, so without making more invasive changes to the class to get access (assuming I'm not missing something that is) the feed item title and link should be sufficient to identify the feed that's getting the fallback, with a little user assistance at the worst.